### ISSUE TITLE
Tweetdeck will constantly log out randomly

### ISSUE THREAD

**Issue by Scoobywya** [dimdenGD/OldTweetDeck#276]
I can like from like 1-2 accounts before it says i no longer have access and it's logging me out to prevent issues.

**Comment by queersplaystuff** [comment]
yeah i've been getting a recurrence of the random lock outs as per back in april, with increasing frequency pretty much since the big domain migration i guess?

I believe it to be similar in cause, too, as my tweetdeck session that has a bunch of search columns is throwing that account, but a separate session for a separate account without any search columns has not experienced the issue.

**Comment by knowthisnumber** [comment]
Yeah, I've been dealing with this for months now. After enough logouts, my account gets suspended. Is there any way that this can be fixed?

**Comment by Scoobywya** [comment]
> Yeah, I've been dealing with this for months now. After enough logouts, my account gets suspended. Is there any way that this can be fixed? Or any steps I can take to avoid getting logged out?

I've noticed if you just go slow and wait for all the buttons/accounts to load on the like list and go slowly with it, it doesn't force logout. Still Would be ideal if we can try to find a fix.

**Comment by dimdenGD** [comment]
Currently the thing that causes most logouts are search columns, it's still not known for sure what causes logouts/suspensions

**Comment by Scoobywya** [comment]
> Currently the thing that causes most logouts are search columns, it's still not known for sure what causes logouts/suspensions

I only have one column which is a searched account, so I wonder if it's just having them in general or if there has to be a different way...

**Comment by knowthisnumber** [comment]
I should add that I've been using OTD on Chrome on a laptop. Could that also be an issue?

**Comment by FirionHope** [comment]
Just got a suspension today for a seemingly nonsense reason Violating our rules against evading suspension." which makes no sense as I haven't been suspended before. Is this the same reason other people are getting from the extension? Has anyone managed to successfully appeal?

**Comment by knowthisnumber** [comment]
> Just got a suspension today for a seemingly nonsense reason Violating our rules against evading suspension." which makes no sense as I haven't been suspended before. Is this the same reason other people are getting from the extension? Has anyone managed to successfully appeal?

Yup, I've been suspended numerous times for the same reason and have been able to successfully appeal each time so far. But this last time I had to appeal multiple times and wait about 3 days for my account to finally be restored (the first few times, it was usually just a 24 hour wait).

**Comment by rjdoerz** [comment]
> Just got a suspension today for a seemingly nonsense reason Violating our rules against evading suspension." which makes no sense as I haven't been suspended before. Is this the same reason other people are getting from the extension? Has anyone managed to successfully appeal?

I've just been suspended as well for the same reason. I've never been suspended before. Don't know how I've ever "evaded" anything. :\

**Comment by queersplaystuff** [comment]
yeah my suspension ended up spreading to my pre-existing other accounts that i wasn't using tweetdeck on, too, all for "evasion" and no appeals are successful

**Comment by eli1382** [comment]
> در حال حاضر چیزی که باعث خروج بیشتر می شود ستون های جستجو هستند، هنوز به طور قطع مشخص نیست که چه چیزی باعث خروج/تعلیق می شود

دیلل آن جستوجوی خودکار یا اسکرول خودکار است

**Comment by eli1382** [comment]
> در حال حاضر چیزی که باعث خروج بیشتر می شود ستون های جستجو هستند، هنوز به طور قطع مشخص نیست که چه چیزی باعث خروج/تعلیق می شود.
The reason is auto search or auto scroll
